SLING-11141 update to o.a.felix.healthcheck.generalchecks 3.x

enapps-enorman opened the pull request:
1. updated to sling-bundle-parent 47
2. updated org.apache.felix.healthcheck.generalchecks dependency to version 3.x

enapps-enorman committed:
SLING-11141 update to o.a.felix.healthcheck.generalchecks 3.x

enapps-enorman committed:
SLING-11141 remove o.a.felix.healthcheck.generalchecks dependency

enapps-enorman committed:
SLING-11141 remove o.a.felix.healthcheck.generalchecks dependency

apelluru commented on pom.xml:
```diff
@@ -23,13 +23,12 @@
-        <artifactId>sling</artifactId>
-        <version>34</version>
+        <artifactId>sling-bundle-parent</artifactId>
+        <version>47</version>
```
in my opinion, we can directly goto version `48`

enapps-enorman replied:
@ashokmca07 Yes, that is probably true, but I would prefer to deal with that later since it is not critical to this PR.  I always try to update to the latest parent version before producing a release candidate but we are not there yet.

enapps-enorman committed:
SLING-11141 add integration test to verify

enapps-enorman committed:
SLING-11141 updated language description

groovy-all bundle doesn't exist after version 2.5 so change the text to
groovy-jsr223

enapps-enorman committed:
SLING-11141 add additional integration tests to verify

Coverage for scriptUrl as local file and as jcr node path

enapps-enorman committed:
SLING-11141 bugfixes and test coverage

use case insensitive language name matching
IT for invalid language name

enapps-enorman committed:
SLING-11141 bugfixes and test coverage

use case insensitive language name matching
IT for invalid language name

enapps-enorman committed:
SLING-11141 improve test coverage

enapps-enorman committed:
SLING-11141 bugfixes and test coverage

added Null Check and try-with-resources to close streams
added tests for better coverage

enapps-enorman committed:
SLING-11141 fix sonar code smell
